packages/cli/src/environments.ee/source-control/__tests__/source-control.service.test.ts

```diff
@@ -139,6 +139,119 @@ describe('SourceControlService', () => {
 	});
 
 	describe('getStatus', () => {
+		it('ensure updatedAt field for last deleted tag', async () => {
+			// ARRANGE
+			const user = mock<User>();
+			user.role = 'global:admin';
+
+			sourceControlImportService.getRemoteVersionIdsFromFiles.mockResolvedValue([]);
+			sourceControlImportService.getLocalVersionIdsFromDb.mockResolvedValue([]);
+			sourceControlImportService.getRemoteCredentialsFromFiles.mockResolvedValue([]);
+			sourceControlImportService.getLocalCredentialsFromDb.mockResolvedValue([]);
+			sourceControlImportService.getRemoteVariablesFromFile.mockResolvedValue([]);
+			sourceControlImportService.getLocalVariablesFromDb.mockResolvedValue([]);
+
+			tagRepository.find.mockResolvedValue([]);
+
+			// Define a tag that does only exist remotely.
+			// Pushing this means it was deleted.
+			sourceControlImportService.getRemoteTagsAndMappingsFromFile.mockResolvedValue({
+				tags: [
+					{
+						id: 'tag-id',
+						name: 'some name',
+					} as TagEntity,
+				],
+				mappings: [],
+			});
+			sourceControlImportService.getLocalTagsAndMappingsFromDb.mockResolvedValue({
+				tags: [],
+				mappings: [],
+			});
+
+			folderRepository.find.mockResolvedValue([]);
+			sourceControlImportService.getRemoteFoldersAndMappingsFromFile.mockResolvedValue({
+				folders: [],
+			});
+			sourceControlImportService.getLocalFoldersAndMappingsFromDb.mockResolvedValue({
+				folders: [],
+			});
+
+			// ACT
+			const pushResult = await sourceControlService.getStatus(user, {
+				direction: 'push',
+				verbose: false,
+				preferLocalVersion: false,
+			});
+
+			// ASSERT
+
+			if (!Array.isArray(pushResult)) {
+				fail('Expected pushResult to be an array.');
+			}
```

Comment:
Do we need this?

Reply:
Yes for type safety, since the method can also return a single object, but this never happens